doc/changes/changes_0.7.0.md

```diff
@@ -0,0 +1,17 @@
+# BucketFs Python {TBD}, released {TBD}
+
+## Summary
+
+TBD
+
+## Documentation
+
+- Fix invalid command in developer guide
+- Update code example in basic user guide
+
+## Changes
+
+- Add project setup section to developer guide
+- Add support exasol-toolbox
+- Add support for pre commit hooks
+- Update dependencies
```

Comment:
ticket ids are missing in this changeslog

Reply:
There haven't been tickets for most of this the only ticket addressed is #7 but this is rather internal I would say